content/user/ui/overview/Basic-quick-start-guide.md

```diff
@@ -0,0 +1,111 @@
+ 
+## Manage Vulnerabilities
+
+Go the Vulnerabilities page to get details on each one and get rid of weaknesses so your application isn't compromised.
+ 
+### Analyze Findings
+
+Contrast discovers any code flaws, which are presented with a severity level to help you prioritize your tasks. For each reported vulnerability, you can mark them and create tags as needed. The following chart shows available statuses and behaviors when a vulnerability is marked and found again.
+
+---
+
+| Status        | Marked                          | Found Again          |
+|---------------|---------------------------------|----------------------|
+| Confirmed     | Stays Open                      | No Change            |
+| Suspicious    | Stays Open                      | No Change            |
+| Not a Problem | Closed - Requires Justification | Stays Closed         |
+| Remediated    | Closed                          | Reopened as **Reported** |
+| Reported      | Default                         | No Change            |
+| Fixed         | Closed                          | Stays Closed         |
+
+---
+
+What’s better than assigning a vulnerability to a user? Creating tags for each vulnerability. These tags can be names of users, groups or just about anything. They are very useful when trying to navigate through vulnerabilities. To create a tag, dive into an application and select the Vulnerabilities tab. The Tag Vulnerabilities option is greyed out until you select the vulnerability you want to tag. 
+
+<a href="assets/images/Tag-vulnerability.png" rel="lightbox" title="Tag vulnerabilities in the application page"><img class="thumbnail" src="assets/images/Tag-vulnerability.png"/></a>
+ 
+> Example: You create **tagA** and assign a few vulnerabilities with it. When you try to browse through your vulnerabilities and want to only look at ones with **tagA**, you can filter for just those.
+
+These tags can be created for applications and servers as well. To learn more about filters, please read [Navigating Vulnerabilities in the New User Interface](https://docs.contrastsecurity.com/user_tsguideapp.html#navvuln). 
+ 
+## Reports
+
+Go to the **Overview** tab in the application page to generate reports of security issues that Contrast identifies while monitoring your application. To learn more, read the article on the [Vulnerability PDF Report](https://docs.contrastsecurity.com/user_tsguiderep.html#pdf).
+ 
+##Track Findings
+
+Contrast gives you the ability to send vulnerabilities to bugtracker integrations or by email for users who don't have access to Contrast. You can set up these and a bunch of other integrations - including Slack, HipChat or any generic WebHook integration - by selecting **Organization Settings** in the User menu and then **Integrations** in the sidebar. You can tell Contrast notify you if there are any new high or critical vulnerabilities found in your application. 
```

Comment:
can we avoid saying things like "a bunch"? How about "you can set up these and many other integrations..."